docs/scripts/generate_plugin_docs.py

```
@@ -993,29 +1021,92 @@ def generate_all_docs(self, readme_content: str):
+            for plugin in self._parse_html_table(section, category):
+                plugins.append(plugin)
+        return plugins
+
+    def generate_all_docs(self, readme_content: str):
+        """Generate all labs documentation."""
+        plugins = self.extract_plugins_from_readme(readme_content)
+        if not plugins:
+            logger.warning("No labs features found in README content")
+            return
+
+        logger.info(f"Found {len(plugins)} labs features")
```

Ruff (0.15.7)

[warning] Logging statement uses f-string

(G004)